extensions/src/tables/index.test.ts

```diff
@@ -0,0 +1,327 @@
+import { createTestSuite } from "$testing";
+import Extension from '.';
+
+createTestSuite({ Extension, __dirname }, {
+  unitTests: {
+    createTable: [
+      (testHelper) => {
+        return {
+          after: async (fixture) => {
+            const { ui, extension, testHelper: { expect, fireEvent, updateHTMLInputValue } } = fixture;
+            const inputs = await ui.findAllByTestId("makeNameInput");
+            const subs = await ui.findAllByTestId("ok");
+            expect(inputs.length).toBe(1);
+            expect(subs.length).toBe(1);
+            
+            const input = inputs[0];
```

Comment:
[BLOCKER] I'm getting some type errors related to these variables `input` and `sub` in this test and below tests as the inputs array is just an `HTMLElement` array, but below you're treating these elements as what they specifically are, like `input` as an `HTMLInputElement`. So just doing an explicit cast should do the trick (and should also make it clear what that thing is), e.g.:

```ts
const input = inputs[0] as HTMLInputElement;
```

I am wondering though -- are you not seeing those errors? Do the tests pass for you?

Reply:
Hmm the tests do pass for me... but i'll do the cast anyways!